cpu/cc13x2/ldscripts/cc1352r1.ld

```diff
@@ -0,0 +1,19 @@
+/**
+ * @addtogroup      cpu_cc13x2
+ * @{
+ *
+ * @file
+ * @brief           linker script for cc13x2 MCUs
+ *
+ * @}
+ */
+
+/* Memory Space Definitions: */
+MEMORY
+{
+    rom (rx)   : ORIGIN = 0x00000000, LENGTH = 0x58000
+    gpram      : ORIGIN = 0x11000000, LENGTH = 8K /* configurable as cache. 20K here, 8K there, and 2K in the ld-script of cc13x2ware */
```

Comment:
What does that comment mean?

Reply:
Honestly, I've just copied it over from cc26x0. I'll take a look into the documentation.

Reply:
So I couldn't find anything about ROM/RAM split, but `gpram` can be configured either as cache memory (default) or as additional 8K of RAM. As `gpram` region is not used elsewhere in RIOT, and reconfiguration support code is also missing, I would suggest just removing this line from both cc13x2  and cc26x0 ld scripts.